interfaces/builtin/x11.go

```diff
@@ -116,8 +118,11 @@ const x11ConnectedPlugAppArmor = `
 # The X abstraction doesn't check the peer label, but in this case that's
 # ok because x11ConnectedSlotAppArmor will limit which clients can connect
 # to the slot implementation.
+# TODO: expand this and remove ICE sockets or add explicit denials for all ICE communications.
```

Comment:
Explicit denials for ICE was done in #9542 - can this comment be removed?

Reply:
Good catch.  I've also removed the second TODO item about access to sockets from a non-implicit x11 slot, since the spread test demonstrates that this works on the systems where AppArmor is enabled.